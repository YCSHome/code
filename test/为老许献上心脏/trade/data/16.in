400 500 500 
319 79442764630247
387 488985694445646
260 85059187651700
186 762468369264594
51 479518041486601
53 611624692500287
159 657467456690308
390 870358367590018
272 697145477783984
49 218352202056090
346 800214206754324
352 538942593564072
247 418749051003605
201 642659267896961
222 772507473689030
150 552374588270639
182 974156042032775
24 925067677935006
229 335631292190890
100 36760543012490
271 408160390829543
242 749135205989455
202 80576541839772
193 325049177013845
245 695816205213856
332 181115377687994
142 225320284497508
88 417988722667433
295 826348138567451
228 255757905045726
316 756270947839622
143 690753994117877
34 584622353059180
351 986460120005569
125 524245991244823
225 1697683114169
215 227810302655170
39 941091045035567
143 449924338252639
54 418146866830324
160 421199995238707
337 534951370777278
305 699133271805763
157 37687258476422
188 744679182243084
188 304118095330875
157 68249261219333
70 501614467216025
41 198665726580433
111 505788796059991
196 848880097634704
95 442558109413318
382 718664387051400
234 83227760819866
82 978289431285325
91 492619231069438
252 405025017646343
136 513679413940003
274 852650564858835
341 980602231224513
240 530084141217682
32 121059224422658
243 797779590147586
51 709767267705647
133 575648593336053
61 449528226345371
244 838758388339438
51 324393527544701
105 412133642916799
45 404112708540374
149 154533276281090
304 469766506125422
378 165620397540164
141 735634745325052
41 315936591180657
257 565136616906119
225 347917690507211
70 250793695619187
214 289400042595873
307 300518531892435
274 824155003830723
57 769129491238130
74 435237561731223
276 773416196960945
9 377371361913418
247 442877110802619
136 219338306155191
28 701051314492307
138 533023582367837
163 920982615538328
8 599545351803125
377 446216394638070
116 714323008437476
129 227907836029652
178 98576241973262
22 545236973067898
328 206216031252940
346 814652155568382
283 420371397760153
133 400314792426480
301 42672408404618
301 928936114087959
273 275994642846057
226 484521369728444
112 109847196052676
130 976685731977874
303 134835017885617
389 950399830045296
8 449698492089111
393 444069115309839
24 440173529552085
7 500633581858136
382 412036220125735
182 691480095223666
198 768038853913279
176 556591243079497
120 180189772261773
145 465306797745433
133 659310352066525
296 276276262016987
206 182683857882059
126 248060752858094
182 974205399134409
370 919264853135071
203 986160787132016
238 63335914834876
120 868860592472393
123 880149517974525
100 740171447962487
94 855504001732359
117 468297126881589
139 994125441751213
353 531518530451820
233 634010768814387
337 642368246560097
23 998599440693763
229 108919915118145
255 701927692463386
51 636424219382229
178 371111125844680
40 796625430694928
50 677575525991880
374 165403779041391
376 722730484024172
392 372738980796754
27 872176692807421
387 300073431682515
78 850154173016548
331 8924458758124
319 56214475782609
37 756450723544770
104 261631054685689
243 116780285440145
334 599714074877687
242 952888399352542
391 600391297423594
298 163991392745390
231 518453650728806
278 443688321346176
380 611395337970251
298 507104579646031
109 352468010223080
360 951729971835290
100 334385297338118
99 20458267618846
175 115842220192154
174 809869532819738
361 679225455579582
3 302748297455277
246 248498157402964
111 517974105405870
316 79708210083658
26 984144550039957
151 888800592073568
48 660355098394154
170 444431369832018
248 716179824479963
250 45416977869807
369 731081937721070
15 724894570989557
274 10186940233360
339 778185447069676
304 975800873167191
351 334950383595578
231 546335155257387
363 508062818136353
395 409646601307275
196 135905037674652
26 121387091728835
300 413073592263060
263 502654294158352
386 36390167408965
211 833378680758765
129 845250235565542
91 951582100417156
172 259886401609148
265 331656143817560
392 85028290600025
369 650019865274134
206 230363975273569
389 493983239845406
17 829558708584477
142 872516827453903
152 166449206567719
363 83183049971477
312 169851104693040
276 350091487975873
27 602599719687688
15 959790384296452
195 938285394953686
66 963921953782419
230 688069124174177
280 853162601497564
13 996493142536113
316 422413969560295
115 560465577387835
14 141700731549202
11 152196412228268
287 79015096238398
30 456977997652961
231 859271283870406
11 812397659834460
259 158097136103738
372 913437917869446
180 882298951944402
167 691658118684776
158 18049320833221
344 183132781740162
178 66990608287370
58 753758637334600
38 464264095658448
92 664888553499331
313 72268610094140
174 346408656401205
130 66235145561775
367 605014848234925
334 131129108921494
150 261783563525463
335 155952614295465
118 962624429816736
315 543156392993128
157 162781762416646
372 245298009385621
143 527857496651338
205 116503424950396
292 489639072465209
257 7966767497032
100 672589031549129
84 416438243109507
238 732833563409340
2 418047846873257
122 210245837293842
69 715713402383780
103 262532507218774
104 773696782029641
371 403237224490633
271 311043783439340
358 969616202556656
85 775370536456514
273 825145113144973
103 319623528409106
274 342256549208034
193 183389964425435
11 407097354391737
280 8734855672104
101 465190549557973
323 367279724271478
157 523206049028886
177 18912807781762
353 580872501061271
56 482016793239227
6 872903060720386
77 46975074709545
288 951851386042549
73 65807667968407
346 898954330579947
102 356974113270876
181 835346927471242
280 67851073681736
61 446331699883794
62 77115483961665
213 951311700659330
336 949352373898057
24 230084632397494
242 259438287685988
65 463091893913327
320 97137700399650
168 699391856369303
87 848131818903742
320 853296009270458
246 36759006756990
159 75914821202127
180 530268607703480
279 621632695119781
346 287302860457637
177 809733235824039
229 924623931261672
305 241861197083700
150 347123342870273
251 122719456916540
217 617422611370092
258 649142807125798
147 740659379167069
154 297688052418198
362 442834783026980
281 968098100047115
255 836076958081801
222 143314661187720
340 430727656127846
162 265875993604358
379 493177507111415
139 291581299208117
361 596367242444978
306 53831450453857
167 812122950380916
373 783235672372985
8 575483992476622
224 138751352743574
68 944490440065678
392 875459088899186
96 533111717145190
327 685686793534432
5 80776572395022
87 221326248473247
383 220268314530696
310 194592326896294
134 656045871309719
376 38394188789376
172 976998302117376
67 806235289422184
339 914056842076086
30 410722857355666
247 822357231655013
102 555894535446184
373 280846013432139
345 895886639996837
118 856918669145814
190 787076724663084
237 571938132057373
400 735436487217551
262 714828278004533
155 975322247756060
129 440946247625401
126 179859659931782
198 473751252866885
142 31739597711095
42 537339832102143
261 301656251486994
320 8474225698654
340 727397379357124
340 628534167031404
303 376544091990914
79 884572261510708
6 423320999184013
175 760980902520709
212 487023104486820
128 898852408626925
99 522599022171959
173 580500395154503
312 926026788238577
279 277408273434479
250 80392553654809
226 674574354512602
3 472271882048617
266 595498671102228
252 711185847679910
170 567782008012056
378 352440714783198
335 792277117267135
201 158549303618942
10 803773946397890
179 767884609225941
299 186146620947887
110 156479690328962
347 838000655451074
203 33358245592333
362 710722168189901
51 428552483913972
199 691284721422568
70 387757345250859
178 980585957164550
291 900757218826837
353 721451628652568
3 172361812587087
275 135927473122793
258 880768013941561
107 491273727762874
399 2297081198880
240 357584785149840
154 725944122690297
269 594584247841981
364 88295782326228
294 992241094935335
321 599343445583392
282 612318855662203
378 34451924897333
394 795936474705385
127 525827342292138
158 950338707612846
369 892502703538139
144 92389025152384
261 182189176972761
97 612367251328641
32 742965454416476
187 381246204461806
251 577720626386427
315 373611745136460
192 225103987693236
34 298017900035188
365 360906544008386
141 464422591230543
56 499764402660609
11 37337054924349
103 389926913999182
180 75276651752045
326 662790050094403
154 438848496526070
6 460056732189082
287 930689616665217
90 545535984456909
90 95451201153028
353 152288069116868
86 683493244417804
340 263560281369213
135 920967366139058
107 690306274994560
201 95059947932850
191 797742222545346
166 12919005317940
269 54865373548519
277 951867064813906
48 438948827603583
75 205142925133974
147 533561744278394
47 685710533289686
71 52495558014521
4 339312158745592
264 240011452590149
115 926548851693589
18 355932107138209
6 777065918762668
190 78824920041278
152 763870959786271
289 724070832413185
1 597653397296878
41 261780052368752
340 988887041879878
273 135237916307395
154 797572819475581
322 179617940513212
198 548962937891092
135 796404613308729
23 213502413501316
286 253286730117318
180 786570964326629
19 819448613087400
149 940799294427306
398 972107975921655
294 201774225405879
94 623668758598833
345 131119377349394
331 299370874777987
340 677844690883097
106 508353829758899
375 462723198544369
29 33380307337131
180 942755023163523
183 810287750916333
295 263696733978570
31 928632289592777
375 42771438983227
379 677793320402414
10 732684548187787
190 861192091469618
355 539084620606914
203 639159585981919
122 346991974865106
150 731042090514574
356 496872783243453
399 327961855075113
374 168348248355292
90 603068886247597
35 625248652678451
260 398442304830898
81 83863135053904
75 526615676148240
112 161790312531982
100 799703671334333
304 310122446567698
195 384013722002608
364 752167582154417
322 984491738206981
66 783137323132902
86 8100657790482
330 980856984019643
177 889971793220138
309 960538245859809
359 977281116608247
181 718654688496923
362 614951600231835
322 786887306876514
264 150746484429513
20 821604875977112
48 624601795498481
233 554929330902192
53 548058572894438
93 966090505890097
167 654977833811618
372 775283732595122
182 665259204041656
181 470817798118423
359 107826183775301
280 790705073581970
238 958856416057617
304 972888313522528
309 676587728740643
341 526665232262220
228 359895183905071
20 242649819739098
339 985270503950594
87 423267601724357
119 389116590980668
102 240773791551273
279 590514159637293
392 589244605252492
352 537177223058491
9 16402862049885
276 320032977181034
21 161781219873121
312 562198510808172
157 129624300424223
236 803468379844796
383 969363760984794
240 568253016973451
222 282505693686782
167 289627168299015
300 620593064932255
54 12639152822509
175 708914459277965
19 412468958616369
350 367810729787673
279 355015844305814
333 104861700872146
106 204181171748621
288 859804374781173
335 956168205130219
124 996210916477468
282 737282325000834
387 938980315265598
147 129505884233389
253 472013891135808
242 267216871708433
279 562149130462017
221 283778904047639
122 450857633801562
298 389741383484312
172 876757517484966
106 474308648752067
79 398384623208882
257 414734831726676
106 440046334455055
396 952938832709176
356 128434076041205
27 916542487787252
303 557828845470492
68 839869020539
326 388788687862932
184 524020516000510
276 53906587334499
237 283072400940732
341 636961568553697
32 162410794825523
137 760704998332420
207 81249754188717
333 85713462195527
136 724276731357270
122 45831935850587
146 724560145695079
248 809087109224873
377 713647766333055
241 188811455692334
302 917477289046822
236 981836289569942
40 508217890787352
217 853788247109568
334 153935115761849
260 387351877918629
267 260576995030783
284 8980276510099
215 232672323040665
227 41951286728642
31 865596555971260
356 314264730857601
194 267132317200328
372 573831358467679
214 109273650953336
120 530793695734117
64 321379494818203
393 178991319975100
236 264308063510580
287 185887791588637
143 110237135607782
390 45130607847104
164 225028579024912
35 335769605227614
126 432759843639900
172 226812167157329
210 679471964665422
343 685727019835226
26 345183692667094
161 596505303231459
186 547280531040220
224 758184426006454
120 323532660301879
118 122853642860935
259 408882378144885
83 733600519512851
173 918820179832091
222 359388280727771
298 336850450126277
22 17340234033363
341 538173576187061
6 588609713558951
177 288738109395766
166 330174995936615
91 391993165824968
382 604964498253253
336 281046661642279
395 872932184786140
265 652916254098316
260 762945484456831
167 112277771639942
51 119038295866358
327 569052449164787
109 17065764140198
182 128012928670654
77 851955103027678
247 548269734515319
331 649038380217263
60 729353655553676
316 897563169664198
195 317557274778302
93 745967216524445
241 453357723108836
306 613684576981987
115 446699285841009
378 999136642422441
126 332791036336614
12 907823244908078
201 376060896559605
302 8327150828697
88 138520679513933
124 414132839685357
63 879259770576385
352 603601722879357
392 113193044814450
252 558155731363980
374 669100253877134
186 404189306758853
142 282869141993070
117 415775704832984
337 958167027800544
89 596689127145343
385 425685315661096
347 363748687084569
306 851630163506574
332 182935596106707
363 885680781621109
151 188756992051605
80 166631102555867
275 126679079597804
264 137619874834193
145 213366097461015
367 498274509042057
352 437297187959612
105 781345671368260
109 292081708299085
32 118226722889840
209 95517651312203
359 390850993958025
120 705560966439179
278 271310647819042
187 658761680394874
50 672569210051461
92 757317640135833
285 93698246244847
10 345707446020033
124 816480397968269
185 513664043101539
44 701156681551773
389 8976546027775
356 560229609460674
284 421589968393612
228 919753179701867
190 830187207204130
400 262958170119044
169 410601551961884
375 272785480762347
288 443960900593956
252 949553261589893
388 61271048255202
13 126580634544471
351 261581453756482
71 178266559983610
23 796674939946251
29 313068454303997
348 100343252505289
315 40116849394923
318 829317008665313
159 959400870892386
332 399493266934399
170 892098384563672
384 869825937421883
383 361050406946889
119 844345527178840
60 509250502927357
384 58372870638376
356 434781101962377
247 167734304890766
54 954062300268683
210 470807872975630
155 929711847503702
185 103883090965830
381 145750431703415
211 674685764349916
16 283057112271939
272 687532099216598
50 186916491817902
324 878156121673662
201 297519598970198
231 966631060661123
2 114484399707221
318 189589407381558
155 874967752465007
352 20338749250995
148 713134195140987
342 24137960318865
159 87988544901851
55 862025315991891
336 187892836639188
377 462110633728392
43 107183868718272
140 545615917393573
133 952073107229177
59 945931737293461
145 819988490707800
222 561627410098304
263 26291828764511
90 983347078489484
300 928418277901320
140 6205516773608
320 954541009808723
263 805892323182775
164 802069612569542
225 327231633477250
13 16225634510415
81 99073365297767
113 244289820623358
77 436288397833093
171 718928145969395
230 400336659752644
296 298883410378907
226 816048190460880
312 894877992596118
128 571739748801217
277 992277772271640
253 709643603530323
165 950573418246372
301 941068003218788
120 523381208173190
212 287310233722834
213 188079864814573
148 411943848694603
297 415747345755697
67 866769131803228
209 825376631573093
147 8569456069469
84 492684859755252
22 423343530686917
127 554887140935704
171 757296905036413
106 877449270168496
247 185756270316043
47 716291029884357
6 462569867372193
134 601308322090005
158 291389572163266
226 457194001802357
41 830948860344717
11 724038122733741
59 236502127966556
205 748795876346169
275 573210343029762
241 325974233026161
277 803246507078251
50 112815997017577
308 139888635321756
55 732592546994606
6 239736004145250
164 910030923947890
