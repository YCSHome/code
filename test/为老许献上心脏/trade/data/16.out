7948411968631197
